lib/parse.js

```diff
@@ -68,7 +68,7 @@ module.exports = (input) => {
     device = _.chain(device)
       .split('\n')
       .filter((line) => {
-        return /^(\s\s-\s)?[a-z]+:/g.test(line);
+        return /^(\s\s[-\s]\s)?[a-zA-Z]+:/g.test(line);
```

Comment:
Ahhh, I think this line is used to filter out any line that _doesn't_ look like YAML, in order to try and prevent e.g. any error messages included in the script-output from breaking the YAML parser.
But according to both http://www.yaml.org/spec/1.2/spec.html#id2759963 and http://www.yamllint.com/ the example in `README.md` of:
```
device: /dev/disk1
description: "Macintosh HD"
size: 249779191808
mountpoints:
  - path: /
raw: /dev/rdisk1
protected: False
system: True
```
would be 'more correct' if formatted as:
```
device: /dev/disk1
description: "Macintosh HD"
size: 249779191808
mountpoints:
  -
    path: /
raw: /dev/rdisk1
protected: False
system: True
```
which I guess means (if the scripts were all updated to output this 'more correct' format) the regex could be updated to `/^((\s{4})?[a-z]+:)|(\s{2}-$)/`. And I guess you could change `volumeName` to something all-lowercase like `label` ;-)

Although having thought about it some more, I'm not sure if adding `label` to `mountpoints` is entirely what you want, as it'll only give you volume-labels for partitions that are actually mounted, and volume-labels for unmounted partitions won't be shown. So I think the discussion in this PR actually overlaps with that in #98